conf/routes

```diff
@@ -4,8 +4,8 @@
 # ~~~~
 
 # An example controller showing a sample home page
-GET        /                    controllers.HomeController.index
-GET        /devices             controllers.DevicesController.getDevices
+GET        /                    controllers.DevicesController.getDevicesHtml()
+GET        /devices.json        controllers.DevicesController.getDevicesJson()
```

Comment:
do you mean adding the .json? I think we can capture it and pass it as a parameter, something like GET        /devices.json        controllers.DevicesController.getDevicesJson()